Updated parameter descriptions in README

YoshuaNava opened the pull request:
I'm updating the parameter descriptions present in the README.

pfankhauser commented on README.md:
```diff
@@ -205,19 +205,24 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 * **`mahalanobis_distance_threshold`** (double, default: 2.5)
 
-    The threshold for the Mahalanobis distance. Decides if measurements are fused with the existing data, overwritten or ignored.
+    Threshold applied for verifying that a new point fits into the elevation map CDF.
```
```suggestion
    Each cell in the elevation map has an uncertainty for its height value. Depending on the Mahalonobis distance of the existing height distribution and the new measurements, the incoming data is fused with the existing estimate, overwritten, or ignored. This parameter determines the threshold on the Mahalanobis distance which determines how the incoming measurements are processed.
```

pfankhauser commented on README.md:
```diff
@@ -205,19 +205,24 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 
-    The threshold for the Mahalanobis distance. Decides if measurements are fused with the existing data, overwritten or ignored.
+    Threshold applied for verifying that a new point fits into the elevation map CDF.
+    The Mahalanobis distance is a point-to-distribution measure that we estimate for each incoming point, and estimates how far, in standard deviations our new measurements can be from the average of all previous measurements.
```
```suggestion
```

pfankhauser commented on README.md:
```diff
@@ -205,19 +205,24 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
-    The threshold for the Mahalanobis distance. Decides if measurements are fused with the existing data, overwritten or ignored.
+    Threshold applied for verifying that a new point fits into the elevation map CDF.
+    The Mahalanobis distance is a point-to-distribution measure that we estimate for each incoming point, and estimates how far, in standard deviations our new measurements can be from the average of all previous measurements.
+    If a point does not fit the current elevation map within the Mahalanobis threshold, it will not be fused with the elevation map, but rather overwritten or ignored.
```
```suggestion
```

pfankhauser commented on README.md:
```diff
@@ -205,19 +205,24 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 * **`multi_height_noise`** (double, default: 9.0e-7)
 
-    Added noise for cell with multiple height measurements (e.g. walls).
+    Noise added to measurements that are higher than the current elevation map at that particular position. This noise-adding process is only performed if a point falls over the Mahalanobis distance threshold.
```
```suggestion
    Noise added to measurements that are higher than the current elevation map at that particular position. This noise-adding process is only performed if a point falls over the Mahalanobis distance threshold. A higher value is useful to adapt faster to dynamic environments (e.g., moving objects), but might cause more noise in the height estimation.
```

YoshuaNava commented:
Hey Péter!
I addressed your comments in the latest commit. Thank you for your feedback.

Additionally, do you want to do something related to the other comments I wrote in the PR description? (unused parameters)
